dockerStart.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# We're going to override the subreddit, comments, and broker if we have an environment variable set
+
+[[ -v SUBREDDIT ]] && echo "posts.subreddits=$SUBREDDIT" >> /kafka-connect-reddit/kafka-connect-reddit-source.properties
```

Comment:
This is a little hacky 🙂
Could we replace with a sed script? Thinking something like:
```suggestion
[[ -v SUBREDDIT ]] && sed -ir 's/^posts\.subreddits=.*$/posts.subreddits='"$SUBREDDIT/" kafka-connect-reddit.properties
```

Reply:
Absolutely.. I was just trying to get a POC working to show it.. I do like the replace more. I'll have to test this out.